src/createThemeFile.ts

```diff
@@ -0,0 +1,51 @@
+import { join, basename } from 'path';
+import * as camelcase from 'camelcase';
+import * as fs from 'fs-extra';
+import WidgetDataInterface from './WidgetDataInterface';
+
+const pkgDir: any = require('pkg-dir');
+const packagePath = pkgDir.sync(__dirname);
+
+async function createThemeFile({
```

Comment:
not sure this needs to be an async function as it currently does not return anything

Reply:
Fixed